react/features/unsupported-browser/components/Landing.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { Platform } from '../../base/react';
+
+import { appNavigate } from '../../app';
+import { landingIsShown } from '../actions';
+
+/**
+ * The map of platforms to URLs at which the mobile app for the associated
+ * platform is available for download.
+ */
+const URLS = {
+    android: 'https://play.google.com/store/apps/details?id=org.jitsi.meet',
+    ios: 'https://itunes.apple.com/us/app/jitsi-meet/id1165103905'
+};
+
+/**
+ * React component representing mobile landing page.
+ *
+ * @class Landing
+ */
+class Landing extends Component {
```

Comment:
Please replace the word landing in all its occurrences (class, Redux state, etc.) with something more appropriate given that the feature is named `unsupported-browser`.